WooCommerce/Classes/ViewRelated/Orders/Order Details/Issue Refunds/OrderRedundsOptionsDeterminer.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+import Yosemite
+
+protocol OrderRefundsOptionsDeterminerProtocol {
+    func determineRefundableOrderItems(from order: Order, with refunds: [Refund]) -> [RefundableOrderItem]
+    func isAnythingToRefund(from order: Order, with refunds: [Refund], currencyFormatter: CurrencyFormatter) -> Bool
```

Comment:
super nit: how about a straightforward name like `isRefundable`?

Reply:
Yeah! I was considering it too because it is more straightforward as you suggest, but at the same time, `isAnythingToRefund` implies that the order is not refundable as a whole but parts of it can be refunded. That helps to avoid confusion like the one I had where I believed that partial orders are not possible 😅